README.md

```diff
@@ -129,6 +130,35 @@ Same button using DI:
 ))
 ```
 
+### Helping users report technical issues
+
+The [hmrcReportTechnicalIssueHelper](src/main/play-26/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/hmrcReportTechnicalIssueHelper.scala.html) component 
```

Comment:
Are we sure we want Helper as a suffix - we are going to end up with a lot of components like this and users are more likely to be wanting to use the helper rather than the underlying component.